src/main/java/com/textmessenger/model/entity/dto/ResponseToFront.java

```diff
@@ -4,7 +4,7 @@
 
 @Data
 public class ResponseToFront {
-  String message;
+  private String message;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "message" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=200&issues=AWYAbmqnM63is4RDlAhA&open=AWYAbmqnM63is4RDlAhA)

[ISSUE_KEY]: AWYAbmqnM63is4RDlAhA